pkgs/development/libraries/amtk/default.nix

```diff
@@ -0,0 +1,40 @@
+{ stdenv, fetchurl
+, pkgconfig, gnome3, dbus, xvfb_run }:
+let
+  version = "4.99.1";
+  pname = "amtk";
+in stdenv.mkDerivation {
+  name = "${pname}-${version}";
+
+  src = fetchurl {
+    url = "mirror://gnome/sources/${pname}/${gnome3.versionBranch version}/${pname}-${version}.tar.xz";
+    sha256 = "00fhvw5y638z584s8cfdslh47ngfzcgx4f0b0456sw8p754j3f8d";
+  };
+
+  nativeBuildInputs = [
+    pkgconfig
+  ];
+
+  buildInputs = [
+    dbus.daemon
```

Comment:
I think this would go to `nativeBuildInputs`, as `dbus-run-session` is run by the build platform. Also the binary is not part of `dbus.daemon` but `dbus`.